src/processes/registerv2-process.js

```diff
@@ -64,16 +65,23 @@ const registerProcess = async (job, done) => {
     if (process.env.AUTO_WLCM_MSG === 'true') {
       await preAutomateWelcomeMsgProcess(returnPrepopulatedDm);
     }
-
+    console.log('::: Start followUser :::');
     await ProcessHelper.followUser(userId, follows);
+    console.log('::: Start createEmptyChannel :::');
     await ProcessHelper.createEmptyChannel(userId, follows);
+    console.log('::: Start followAnonymousUser :::');
     await ProcessHelper.followAnonymousUser(anonUserId, follows);
+    console.log('::: Start followTopic :::');
     await ProcessHelper.followTopic(userId, topics);
+    console.log('::: Start followLocation :::');
     await ProcessHelper.followLocation(userId, result?.locations);
+    console.log('::: Start sendTopicAutoMessage :::');
     await ProcessHelper.sendTopicAutoMessage(userId, topicIds);
-
+    console.log('::: Start followMainFeedTopic :::');
     await ProcessHelper.followMainFeedTopic(userId, topicNames);
+    console.log('::: Start followMainFeedFollowing :::');
     await ProcessHelper.followMainFeedFollowing(userId, follows);
+    console.log('::: Start addUserToTopicChannel :::');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While adding logging statements can be useful for debugging, it's important to consider the potential performance implications. Logging in JavaScript is a blocking operation, and if these operations are frequent, they could slow down your application. If you need these logs for debugging purposes, consider using a logging level environment variable to control when these logs should be output.

```diff
-     console.log('::: Start followUser :::');
+     if (process.env.LOG_LEVEL === 'debug') console.log('::: Start followUser :::');
      await ProcessHelper.followUser(userId, follows);
-     console.log('::: Start createEmptyChannel :::');
+     if (process.env.LOG_LEVEL === 'debug') console.log('::: Start createEmptyChannel :::');
      await ProcessHelper.createEmptyChannel(userId, follows);
-     console.log('::: Start followAnonymousUser :::');
+     if (process.env.LOG_LEVEL === 'debug') console.log('::: Start followAnonymousUser :::');
      await ProcessHelper.followAnonymousUser(anonUserId, follows);
-     console.log('::: Start followTopic :::');
+     if (process.env.LOG_LEVEL === 'debug') console.log('::: Start followTopic :::');
      await ProcessHelper.followTopic(userId, topics);
-     console.log('::: Start followLocation :::');
+     if (process.env.LOG_LEVEL === 'debug') console.log('::: Start followLocation :::');
      await ProcessHelper.followLocation(userId, result?.locations);
-     console.log('::: Start sendTopicAutoMessage :::');
+     if (process.env.LOG_LEVEL === 'debug') console.log('::: Start sendTopicAutoMessage :::');
      await ProcessHelper.sendTopicAutoMessage(userId, topicIds);
-     console.log('::: Start followMainFeedTopic :::');
+     if (process.env.LOG_LEVEL === 'debug') console.log('::: Start followMainFeedTopic :::');
      await ProcessHelper.followMainFeedTopic(userId, topicNames);
-     console.log('::: Start followMainFeedFollowing :::');
+     if (process.env.LOG_LEVEL === 'debug') console.log('::: Start followMainFeedFollowing :::');
      await ProcessHelper.followMainFeedFollowing(userId, follows);
-     console.log('::: Start addUserToTopicChannel :::');
+     if (process.env.LOG_LEVEL === 'debug') console.log('::: Start addUserToTopicChannel :::');
      await ProcessHelper.addUserToTopicChannel(userId, topicNames);
```
This way, you can control the logging level from your environment and avoid unnecessary log statements in a production environment.


<!-- This is an auto-generated comment by OSS CodeRabbit -->